test/index.spec.ts

```diff
@@ -132,16 +128,30 @@ const getRuleName = (configuredRule: string): string => {
  */
 describe.each(Object.keys(configuredRules))(
   '%s rules',
-  (ruleSet: string): void => {
-    describe.each(Object.keys(configuredRules[ruleSet].rules))(
+  async (ruleSet: string): Promise<void> => {
+    const ruleSetFixtures = await fixturesPromises[`${ruleSet}Fixtures`];
```

Comment:
apparently even in the previous tests version, the idea was to have fixtures for every rule that has a custom configuration in `rules` folder (makes sense). However, it looks like it wasn't set up correctly with Jest, as you can see not every rule configured in `rules/*` currently has fixtures.

vitest doesn't allow empty test suites and considers them failed, so I had to filter out the rules that don't have fixtures or are turned off slightly differently, but otherwise no changes: you can see on CI that the same 288 tests are passing